test/core/miner/precommit_batcher_test.cpp

```diff
@@ -117,18 +159,26 @@ namespace fc::mining {
     si.sector_number = 2;
 
     EXPECT_OUTCOME_TRUE_1(batcher_->addPreCommit(
-        si, deposit, precInf, [](const outcome::result<CID> &cid) {
+        si,
+        deposit,
+        precInf,
+        [](const outcome::result<CID> &cid) -> outcome::result<void> {
           EXPECT_OUTCOME_TRUE_1(cid);
-          ASSERT_TRUE(cid.has_value());
+          EXPECT_TRUE(cid.has_value());
```

Comment:
Redundant check, result must have a value, if it is successful at the line above.

Reply:
I think, this lambda may be written as function `SelectAddressMock()` because it is used several times.

Reply:
this isn't related to address selector module, this is callback function for checking the results of batched sendings, this lambdas helpful for additional attempts of proving in case of failure